lib/ansible/parsing/vault/__init__.py

```diff
@@ -736,7 +736,7 @@ def decrypt_and_get_vault_id(self, vaulttext, filename=None):
                     vault_secret_used = vault_secret
                     file_slug = ''
                     if filename:
-                        file_slug = ' of "%s"' % filename
+                        file_slug = ' of "%s"' % to_text(filename)
```

Comment:
__Unsure__ but shouldn't you use `u""` prefix to prevent merging bytes and unicode (on Python2). @webknjaz ?